plugin/src/App/StatusPages/Settings/init.lua

```diff
@@ -104,6 +104,14 @@ function SettingsPage:render()
 					layoutOrder = layoutIncrement(),
 				}),
 
+				SyncLock = e(Setting, {
+					id = "syncLock",
+					name = "Sync Lock",
+					description = "Toggle sync lock",
```

Comment:
This description is not clear at all, and doesn't explain any more than the title already says.
```suggestion
					description = "Enables Team Create sync locking to prevent conflicts and overwrites between multiple users.",
```